src/db/ClimbTypes.ts

```diff
@@ -27,21 +27,35 @@ export type ClimbGQLQueryType = ClimbType & {
  * Clinbs have a number of fields that may be expected to appear within their documents.
  */
 export type ClimbType = IClimbProps & {
+  pitches?: IPitch[]
   metadata: IClimbMetadata
   content?: IClimbContent
 }
 
+/* Models a single pitch of a multi-pitch route */
+export interface IPitch {
+  _id: MUUID
+  parentId: string
```

Comment:
Can we change `IPitch.parentId` to MUUID?  Since it references `Climb._id` which has MUUID type.